frontend/javascript/components/publicbody/pb-filter-list.vue

```diff
@@ -13,10 +13,11 @@
             @change="setFilter(item)"
             :data-test="item.isActive"
             :checked="item.isActive" />
-          <label class="form-check-label" :for="item.labelId">
+          <label class="form-check-label text-break" :for="item.labelId">
             {{ item.label }}
+            <small v-if="item.subLabel" class="text-secondary d-block">{{ item.subLabel }}</small>
           </label>
-          <small v-if="item.count">({{ item.count }})</small>
+          <small v-if="item.count" class="filter-list-sublabel">({{ item.count }})</small>
           <i
```

Comment:
With public body categories, the subcategories show up twice: Once collapsed with the parent, and then below on their own, prefixed by a dash.

<img width="277" height="461" alt="image" src="https://github.com/user-attachments/assets/e8361ccd-6a34-43be-893a-483a301db7c4" />